virtwho/executor.py

```diff
@@ -181,6 +183,17 @@ def run_oneshot(self):
             thread.start()
 
         Executor.wait_on_threads(self.destinations)
+        if self.options[VW_GLOBAL]['status']:
+            output = {}
+            for source in self.dest_to_source_mapper.sources:
+                try:
+                    report = self.datastore.get(source)
```

Comment:
I guess that only this line`report = self.datastore.get(source)` is reason for try-except. I would move following two lines to `else` block. It is good practice to have only one line between try-except.